plugins/modules/web_infrastructure/bugzilla.py

```diff
@@ -0,0 +1,216 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, Chuang Cao <chcao@redhat.com>
+# Atlassian open-source approval reference OSR-76.
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = """
+module: bugzilla
+version_added: "1.0.0"
+short_description: Manage bugs in a Bugzilla instance by xmlrpc
+description:
+  -  Fetch, comment or change status on bugs of a Bugzilla instance.
+
+options:
+  uri:
+    required: true
+    description:
+      - Base URI of the Bugzilla instance.
+    type: str
+
+  operation:
+    required: true
+    type: str
+    choices: [ comment, transition, fetch ]
+    aliases: [ command ]
+    description:
+      - The operation to perform.
```

Comment:
You need to document what the three actions actually do.

One question about fetch: is this not posting anything, but fetching information on the bug? Then it should not be a command of this module, but a separate bugzilla_info module.

Reply:
It needs to send the bug id to the bugzilla, the function works well on my local env, so I don't want to do the further change.

>> You need to document what the three actions actually do.
The rest of the document mentions these operations, but yeah, I will add the description here.

Reply:
It's not really optional, see the part on `Do not add get, list or info state options` in https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_best_practices.html#scoping-your-module-s

Reply:
Do not add get, list or info state options to **an existing module** - create a new _info or _facts module.

Thanks a lot for pointing that, I can see it mentioned we shouldn't add the get/list/info into existing module, but this is a fresh new module. I think it's acceptable. What's your thinking?

Reply:
agreed with @felixfontein 
@xiangge please move the `fetch` functionality to a separate module